consent/models.py

```diff
@@ -26,17 +28,35 @@ class DataSharingConsentQuerySet(models.query.QuerySet):
     except is not saved in the database until committed.
     """
 
-    def get(self, *args, **kwargs):
+    def proxied_get(self, *args, **kwargs):
```

Comment:
Yeah, I'd be open to suggestions, but I don't have anything better off the top of my head.